qlib/strategy/base.py

```diff
@@ -79,28 +84,30 @@ def reset(
         self,
         level_infra: LevelInfrastructure = None,
         common_infra: CommonInfrastructure = None,
-        outer_trade_decision=None,
-        **kwargs,
-    ):
+        outer_trade_decision: BaseTradeDecision = None,
+        **kwargs,  # TODO: remove this?
```

Comment:
If you confirm Yuge's example does not need this, you can remove it.

Reply:
Just keep if for now is better. I'll delete this comment.